templates/base.html

```diff
@@ -4,7 +4,6 @@
     {% block head_early %}{% endblock %}
     <title>{% if title %}{{ title }} - {% endif %}{% if banner %}{{ banner }} - {% endif %}Gratipay</title>
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta name="referrer" content="no-referrer-when-downgrade">
```

Comment:
- switch from [`no-referrer-when-downgrade`](https://w3c.github.io/webappsec-referrer-policy/#referrer-policy-no-referrer-when-downgrade) to [`strict-origin-when-cross-origin`](https://w3c.github.io/webappsec-referrer-policy/#referrer-policy-strict-origin-when-cross-origin)
- set header instead of using `meta`
- test it :)

Am I reading this right?